src/test/java/uk/gov/hmcts/reform/bulkscanning/controller/PaymentControllerTest.java

```diff
@@ -145,7 +145,7 @@ public void testCreatePaymentForBulkScan() throws Exception{
        when(paymentService.saveInitialMetadataFromBs(any(BulkScanPaymentRequest.class)))
            .thenReturn(Arrays.asList(dcn));
 
-       ResultActions resultActions = mockMvc.perform(post("/bulk-scan-payments/")
+       ResultActions resultActions = mockMvc.perform(post("/bulk-scan-payments")
```

Comment:
Pay attention to this potentially breaking change. In V2 those two urls were treated the same, in V3, they are different and therefore we need to be explicit if we want to handle urls ending with `/` the same as without slash.

Reply:
yes, this is correct implementation, I have seen this behaviour on Ref Data

Reply:
What I mean breaking is that if clients submitting with `/` at the end of url, with spring boot 2, it simply worked, with the upgrade, it is going to break for them.

Reply:
Thanks @rapolaskaseliscgi POST /bulk-scan-payments is the correct format.